Help-Faster-Builds.txt

```diff
@@ -0,0 +1,51 @@
+CLIP builds take some time. A good bit of this time
+is due to re-rolling an entire installation image
+including re-creating Anaconda's live environment
+used during installation. One way to think of it is
+
+1. CLIP builds source packages, then binary packages.
+
+2. CLIP creates does a full install of all the packages
```

Comment:
What does "CLIP creates does a... " mean?

Reply:
I'll reword in the next push.